Writerside/topics/raptor-xx.md

```diff
@@ -1,3 +1,435 @@
-# RaptorXX
+# Simple C++ RAPTOR Implementation
 
-Implementation of the simple Raptor in C++ for performance comparison in benchmarking.
+This section discusses the design, implementation, and optimization of a public transport routing system, developed
+using modern C++20/C++23 standards. The project incorporates advanced data structures and algorithms to enable
+efficient route planning, with an emphasis on cross-platform compatibility and robust build configurations via CMake.
+
+The C++ implementation is based heavily on the [Simple Raptor](simple-raptor.md "Simple RAPTOR Implementation"), but
+extends it with optimizations and modern C++ features. In this section, we focus on the C++ implementation, detailing
```

Comment:
how does it extend the Java simple raptor?

Reply:
That was meant more in the sense that it adopts the same logic.